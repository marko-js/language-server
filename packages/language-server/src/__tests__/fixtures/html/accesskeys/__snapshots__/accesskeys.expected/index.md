## Diagnostics
### Ln 1, Col 2
```marko
> 1 | <a href="google.com" accesskey="g">Link to Google</a>
    |  ^ Fix all of the following:
  Document has multiple elements with the same accesskey
  2 | <a href="github.com" accesskey="g">Link to GitHub</a>
```

