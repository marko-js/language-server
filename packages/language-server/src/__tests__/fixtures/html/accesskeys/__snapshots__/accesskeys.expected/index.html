<a data-marko-node-id="0" href="google.com" accesskey="g">Link to Google</a><a data-marko-node-id="1" href="github.com" accesskey="g">Link to GitHub</a>